db/migrate/20240421080016_create_users.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def add_unique_constraints(table_name, columns)
+    columns.each do |column|
+      execute <<-SQL
+        ALTER TABLE #{table_name}
+        ADD CONSTRAINT check_if_#{column}_is_unique
+        UNIQUE (#{column});
+      SQL
+    end
+  end
+
+  def up
+    create_table :users do |t|
+      t.string :first_name, limit: 128
+      t.string :last_name, limit: 128
+      t.integer :gender, default: 0
```

Comment:
@kth-wtag apu, understood and done. However, is it because this is a medical appointment scheduler thus gender should be a crucial data or should gender be a mandatory field in general?